### ISSUE TITLE
JSON-LD export does not round trip through Import with repeated nodes

### ISSUE THREAD

**Issue by azaroth42** [archesproject/arches#4563]
**Describe the bug**

Due to the framing algorithm used to turn the graph into JSON-LD, when a node in the graph appears multiple times, such as when a Concept is used in multiple places, the full information is only serialized once.

So the export comes out as:

```
"dimension":  [
{
  "classified_as": [{"id": "http://vocab.getty.edu/aat/300055647", "label": "width", "type": "Type"}], 
  "id": "http://localhost:8000/tile/84153f9b-60b6-4a87-aefe-a5d57a60b1f0/node/1e5ddc21-2e44-11e9-9c6a-a4d18cec433a", 
  "type": "Dimension",
  "unit": "http://vocab.getty.edu/aat/300379098", 
  "value": 20}, 
{
  "classified_as": [{"id": "http://vocab.getty.edu/aat/300055644", "label": "height", "type": "Type"}], 
  "id": "http://localhost:8000/tile/deb515ff-a9e9-437e-bfb6-a11af32cbbfc/node/1e5ddc21-2e44-11e9-9c6a-a4d18cec433a", 
  "type": "Dimension", 
  "unit": {"id": "http://vocab.getty.edu/aat/300379098", "label": "centimeters", "type": "MeasurementUnit"}, 
  "value": 10}
],
```
But the import expects the `unit` to have the same construction in both cases.

This matters if you want to take data out and put it back in again, or if you are writing an API consumer that expects it always to have the same construction.  Both of which we intend to do, but not next week :)

In the JSON-LD 1.1 working group, this may be solved by at least one of three feature requests:

* https://github.com/w3c/json-ld-framing/issues/29 
* https://github.com/w3c/json-ld-api/issues/33
* https://github.com/w3c/json-ld-syntax/issues/19 

At which point the frame can be written and the development cost handed off to the PyLD library.

**Comment by azaroth42** [comment]
The feature in JSON-LD 1.1 was deferred for a future version :( Still a problem but unlikely to be solved without some explicit customization of the JSON after producing it from the graph.

**Comment by apeters** [comment]
fixed by #7546
